app/Providers/NovaServiceProvider.php

```diff
@@ -72,6 +73,11 @@ protected function gate(): void
      */
     protected function cards(): array
     {
+        if ('/nova-api/dashboards/main' !== $_SERVER['REQUEST_URI'] &&
+            ! Str::startsWith($_SERVER['REQUEST_URI'], '/nova-api/metrics/')) {
```

Comment:
There's not a Laravel way to get that variable?

Reply:
Maybe, let me try something